File: laptop/C_Solution/단계별/기본수학1/10250.c
// ACM Hotel

#include <stdio.h>

int main(){
    int t;
    for(scanf("%d", &t); t > 0; t--){
        int h,w,n;
        scanf("%d %d %d", &h, &w, &n);

        int y, x;
        if(n % h == 0) y=h;
        else y=n%h;
        x = (n-y)/h+1;

        printf("%d%02d\n", y, x);
    }
    return 0;
}

File: laptop/Python_Solution/단계별/기본수학1/10250.py
# ACM Hotel


h,w,n = map(int,input().split())
y=h if n%h==0 else n%h
x=(n-y)/h+1
print("%d%02d"%(y,x))

#exec('h,w,n=map(int,input().split());y=h if n%h==0 else n%h;print("%d%02d"%(y,(n-y)/h+1));'*int(input()))